src/main/java/se/kth/meta/entity/User.java

```diff
@@ -49,13 +49,11 @@ public int getId() {
 
   @Override
   public void valueBound(HttpSessionBindingEvent event) {
-
-    System.out.println("USER " + username + " WAS BOUND TO THE SESSION");
+    //System.out.println("USER " + username + " WAS BOUND TO THE SESSION");
```

Comment:
What's the use of this class? It's not an entity, so it probably should be in a different package. Also, don't comment out code you don't need anymore; just remove it. Documentation as to why the method is empty, what the method does,... is desirable.